src/to-have-style.js

```diff
@@ -0,0 +1,38 @@
+import {matcherHint} from 'jest-matcher-utils'
+import {checkHtmlElement, getMessage} from './utils'
+
+function parseCSS(css) {
```

Comment:
You might consider using the `css` module for this.

Reply:
Yes, I will. It's on the to-do list above. This was just a first draft, a proof-of-concept.